src/components/Contacts/ContactRow.tsx

```diff
@@ -8,13 +8,29 @@ import { GiftStatus } from './GiftStatus/GiftStatus';
 import { StarContactIcon } from './StarContactIcon/StarContactIcon';
 
 const useStyles = makeStyles((theme: Theme) => ({
+  contactRowButton: {
```

Comment:
Thanks for cleaning this up 🙂 Do you think in your next pr we should migrate this away from ```makeStyles``` in favor of the ```styled``` API?

Reply:
I wasn't sure if `styled` would work for the Material UI icon in this component... would it work?  If so then I have no problem moving to `styled`.

Reply:
Hmm, I haven't specifically tested it on a material UI icon, but I would say it should work 🙂